pvlib/tests/spectrum/test_mismatch.py

```diff
@@ -302,3 +302,107 @@ def test_spectral_factor_jrc_supplied_ambiguous():
     with pytest.raises(ValueError, match='No valid input provided'):
         spectrum.spectral_factor_jrc(1.0, 0.8, module_type=None,
                                      coefficients=None)
+
+
+@pytest.fixture
+def sr_and_eqe_fixture():
```

Comment:
I think the changes in this file may be unintentional -- should they be reverted?